site/community.html

```diff
@@ -0,0 +1,38 @@
+---
+layout: page
+title: Community
+description: Sonobuoy Community
+id: community
+---
+
+<div class="sub-page">
+    <h2>Do you want to help build Sonobuoy?</h2>
```

Comment:
@amymanion shouldn't this be h1?

Reply:
@ccbayer No, there is a "Community" header at the top of the page which is the h1.